src/Kunstmaan/PagePartBundle/PagePartAdmin/AbstractPagePartAdminConfigurator.php

```diff
@@ -2,24 +2,7 @@
 
 namespace Kunstmaan\PagePartBundle\PagePartAdmin;
 
-/**
- * AbstractPagePartAdminConfigurator
- */
-abstract class AbstractPagePartAdminConfigurator
+abstract class AbstractPagePartAdminConfigurator implements PagePartAdminConfiguratorInterface
```

Comment:
Since this abstract class has been made completely empty, wouldn't it make more sense to just completely remove it all together? Just implement the interface directly where you would otherwise extend this class.